src/index.js

```diff
@@ -1,10 +1,26 @@
 import './style.css';
 import { scores } from './modules/data.js';
+import { fetchData, createData } from './modules/actions.js';
+import { showAlert } from './modules/utils.js';
 
 const list = document.querySelector('.list');
-list.innerHTML = '';
-scores.forEach((score) => {
-  list.innerHTML += `
-           <p>Name: ${score}</p>
-    `;
-});
+const submitButton = document.querySelector('.btn-submit');
+const refreshButton = document.querySelector('.btn-refresh');
+const user = document.querySelector('#playerName');
+const score = document.querySelector('#score');
+
+refreshButton.addEventListener('click', () => window.location.reload());
+
+const init = () => {
+  submitButton.addEventListener('click', (e) => {
+    e.preventDefault();
+    if (user.value !== '' && score.value) {
+      createData(user.value, score.value);
+    } else {
+      showAlert('All values are required');
+    }
```

Comment:
- [ ] [Optional]For a better user experience, you should set back `user.value` and `score.value` back to `' '` after form submit